compiler/fir/java/src/org/jetbrains/kotlin/fir/java/deserialization/JvmBinaryAnnotationDeserializer.kt

```diff
@@ -118,7 +170,8 @@ class JvmBinaryAnnotationDeserializer(
                         if (signature.hasGetter()) MemberSignature.fromMethod(nameResolver, signature.getter) else null
                     CallableKind.PROPERTY_SETTER ->
                         if (signature.hasSetter()) MemberSignature.fromMethod(nameResolver, signature.setter) else null
-                    // TODO: PROPERTY
+                    CallableKind.PROPERTY ->
+                        getPropertySignature(proto, nameResolver, typeTable, true, true, requireHasFieldFlagForField)
```

Comment:
no need to pass the last two arguments when field is true?

Reply:
Thank you for spotting this! You're right; if `field` is true, `synthetic` is not necessary. Added https://github.com/JetBrains/kotlin/pull/3493/commits/d86d3b8d8f46dc67da10e0e4c105744d61e5b104 to change FE 1.0 part as well.